examples/ResourceGroup/ResourceGroup.Tests.ps1

```diff
@@ -1,5 +1,4 @@
 ﻿BeforeAll {
-  Import-Module BenchPress.Azure
```

Comment:
We should either replicate this removal from all the examples, or leave it here.
I'm all for removing it.

Reply:
I'll add it back in and in a separate PR we can remove them all.

Reply:
Added back